medicines/doc-index-updater/src/storage_client/azure_blob_client.rs

```diff
@@ -75,12 +74,13 @@ impl StorageClient for AzureBlobStorage {
     async fn add_file(
         &self,
         file_data: &[u8],
+        license_number: &str,
```

Comment:
This is quite a strong opinion for the `storage_client` to have: previously, it only new it was handling files with metadata, not that there'd be anything like a `license_number` relevant for that file.

Reply:
Yeah, good point. We can probably adjust this interface to be a bit cleaner somehow